src/avkit.cs

```diff
@@ -95,7 +95,7 @@ interface AVPictureInPictureControllerDelegate
 	}
 	
 
-	[iOS (8,0)]
+	[iOS (8,0), TV (9,0)]
```

Comment:
We do have the class under /Applications/Xcode73.app/Contents/Developer/Platforms/AppleTVOS.platform//Developer/SDKs/AppleTVOS9.2.sdk/System/Library/Frameworks/AVKit.framework/Headers/AVPlayerViewController.h with a NS_CLASS_AVAILABLE_IOS (9_0)